### PR TITLE
60 67

### PR DESCRIPTION
**Pull request by InfamousVague**
### What this PR does 📖
- 

### Which issue(s) this PR fixes 🔨
- Resolve #

### CODE REVIEW

File: src/components/prelude/auth/mod.rs [hunk 1]
```diff
@@ -19,6 +20,11 @@ pub struct Props {
     account: Account,
 }
 
+fn remove_whitespace(s: &mut String) -> String {
```
Comment (dariusc93) [inline src/components/prelude/auth/mod.rs]: If youre going to use this you should either mutate the borrowed string or return. Not both. eg

```rust
fn remove_writespace(s: &mut String) {
    s.retain(|c| !c.is_whitespace());
}
```

or 

```rust
fn remove_writespace(s: String) -> String {
    let mut s = s.clone();
    s.retain(|c| !c.is_whitespace());
    s
}
```

Might be better to use `&str` for the latter

**Review by dariusc93 — COMMENTED** [review]
Made a small comment here

**Comment by phillsatellite** [comment]
Added Changes Request for comment above from Darius 👍

**Comment by stavares843** [comment]
what are you doing here 😂

**Comment by InfamousVague** [comment]
Making it so usernames cannot have spaces in them

**Comment by stavares843** [comment]
weekend @phillsatellite 😂

**Pull request merged**